cypress/integration/loggedOutFlows/runtimeContext.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Runtime Context', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should detect the current context and set data-runtime attr', () => {
+    // Visit the site
+    cy.visit('/');
+
+    // Check the body's data-runtime attribute
+    cy.get('body').should('have.attr', 'data-runtime');
```

Comment:
This test is very barebones. I originally wanted to make sure the attribute was set accordingly to the `navigator.platform` and `navigator.userAgent` values but I wasn't able to stub them. If this is okay we can carry on with this, otherwise it would be awesome if I get some help to make this work in Cypress 🙂

Reply:
I had a little look and it seems like we can stub them like this:

```js
 cy.visit('/', {
      onBeforeLoad: (win) => {
        Object.defineProperty(win.navigator, 'userAgent', {
          value: 'ForemWebView',
        });
        Object.defineProperty(win.navigator, 'platform', { value: 'Windows' });
      },
    });

    // Check the body's data-runtime attribute
    cy.get('body')
      .should('have.attr', 'data-runtime')
      .and('equal', 'ForemWebView-Windows');
```

It worked for me locally 🙂

Reply:
I tried a few different ways using some parts of this suggestion but never really made it work. This works as a charm though, thank you @aitchiss! 

I just added as many as I could (i.e. we don't support `ForemWebView` in Android yet). Let me know if you think we should approach it differently 🙂